linalg/sundials.cpp

```diff
@@ -2053,6 +2053,10 @@ void KINSolver::SetOperator(const Operator &op)
          MFEM_ASSERT(flag == KIN_SUCCESS, "error in KINSetMAA()");
       }
 
+      // Set Anderson acceleration QR orthogonalization method
+      flag = KINSetOrthAA(sundials_mem, orthaa);
+      MFEM_ASSERT(flag == KIN_SUCCESS, "error in KINSetOrthAA()");
```

Comment:
If I understand this correctly, not calling `KINSetOrthAA` will use whatever the default method in SUNDIALS is.  It is currently `KIN_ORTH_MGS`, which is why this block of code is not inside a conditional, similar to the number of acceleration vectors above.  If this is all correct, then my concern is this approach effectively redefines the default method in MFEM.  I suggest instead to only call `KINSetOrthAA` if the user has called `KINSolver::SetOrthAA`.

Reply:
No, the variable is initialized in the constructor initializer list with `orthaa(KIN_ORTH_MGS)`.

Reply:
Doesn't that effectively define the default behavior in MFEM to use `KIN_ORTH_MGS`?

Reply:
Yes, which is already the default. Sundials has this funny way of saying "if the subspace size is 0, we don't do acceleration". So there are 10 knobs to turn to get a simple "on/off" switch. Which points me to the real issue, you can't set the subspace yet 👎